source/game_preparation/calibration/calibration_getup.rst

```diff
@@ -0,0 +1,98 @@
+###############
+Getup
+###############
+
+We fall over a lot, so we need to make sure the getup works reliably.
+
+************
+Instructions
+************
+
+#.  Find the most inclined part of the field of the next game.
+#.  Run `runswift` (triple-tap), stiffen (double-tap), and put the robot in the penalised state (double single-tap).
```

Comment:
The `runswift` executable is usually running after a reboot, so that bit isn't really needed. If we're being super-explicit then changing field away from any GameControllers (which will override the chest button penalised/playing) could be useful.

```suggestion
#.  Stiffen (double-tap) and put the robot in the penalised state (double single-tap).
```

Reply:
> The `runswift` executable is usually running after a reboot

oh has that changed?  are we still using daemon.py for anything?

> changing field away from any GameControllers

should definitely specify this.  i was only thinking of competition conditions where the robot isn't on a network during calibration.  can also add something about the fastest way to iterate.  taking your laptop to the field and connecting a network cable is not a good idea as 1) it affects the getup, 2) holding the laptop and catching the robot at the same time is hard.  also if your laptop isn't set up with a dhcp server, falling back to link local takes time.  so bring a router close to the field and leave your laptop and a cable connected to it.  then move the robot back and forth between the cable and the inclined spot on the field.  best if you have two people (one on laptop and one handling the robot).